### ISSUE TITLE
Multiattacks not working with D&D 1.6.1

### ISSUE THREAD

**Issue by 346184** [jessev14/Multiattack-5e#11]
I recently updated to D&D 5E system version 1.6.1 and the following error occurs when rolling multiple attacks using the module, regardless if its for PCs or NPCs and no rolls occur, only the error in console:
```
Uncaught (in promise) Error: The provided data is not an object!
[Detected 3 packages: multiattack-5e, system:dnd5e, more-hooks-5e]
    at isObjectEmpty (helpers.mjs:428:11)
    at ma5e.js:355:42
    at ma5e.js:387:31
    at Hooks._call (foundry.js:294:14)
    at Hooks.call (foundry.js:279:33)
    at ClientDatabaseBackend._preCreateDocumentArray (foundry.js:10013:47)
    at async ClientDatabaseBackend._createDocuments (foundry.js:9930:22)
    at async ChatMessage.createDocuments (document.mjs:338:21)
    at async ChatMessage.create (document.mjs:439:21)
    at async d20Roll (dice.js:263:30)
    at async Item5e.rollAttack (entity.js:1180:18)
    at async Item5e.rollAttackPatch (rollAttack.js:8:18)
    at async Item5e._onChatCardAction (entity.js:1561:9)
```
Non-multiattack attack rolls work fine.

This is with latest Multiattack-5e version 6.2.0. I love the module, so a fix would be welcome if possible :) If I can provide any more to help, I'm happy to help!

**Comment by jessev14** [comment]
are you using MRE?

**Comment by zerastos** [comment]
@jessev14 Yes, I am :) The combination did work before the update though!

**Comment by jessev14** [comment]
Okay, I mostly fixed MRE compatibility. The only thing that doesn't work is if you have multiple formula groups set up on an item, Multiattack won't really work, it'll roll with the primary roll group. I've put in a pull request to address this on MRE's side, but it's up to them if they want to implement it.

**Comment by zerastos** [comment]
Thanks so much! For future readers, the PR on MRE's side is https://github.com/ElfFriend-DnD/FVTT-Minimal-Rolling-Enhancements-DND5E/pull/45 and it got merged too.
Awesome, thanks for the quick fix and for working on these modules, really making my sessions a breeze :)

**Comment by zerastos** [comment]
Actually, there is one more thing: for attack rolls only, it seems the original roll is no longer supressed?
I.e. if using Multiattack, there is always the original roll, then the multiattack ones, so one more than requested.
A single (non-multiattack) roll works as it should.

This doesn't seem to be occuring for damage rolls.
This also doesn't seem to be occuring with MRE disabled.
This is with the most recent updates to MRE (the PR above) and Multiattack.

**Comment by jessev14** [comment]
oops. fixed in 6.2.3.
